packages/charts/src/styles/vaadin-chart-base-styles.js

```diff
@@ -149,1048 +149,1248 @@ export const chartStyles = css`
     width: 100%;
     height: 100%;
     text-align: left;
-    z-index: 0;
+    line-height: normal;
+    z-index: 0; /* highcharts/highcharts#1072 */
     -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+    user-select: none;
+    touch-action: manipulation;
+    outline: none;
   }
 
-  :where([styled-mode]) .highcharts-root {
-    display: block;
-  }
+  :where([styled-mode]) {
```

Comment:
Not sure if it's worth it, but we could make reviewing easier by extracting CSS nesting change into separate PR so that this one would only contain actual changes to styles.